website/docs/getting-started/setup/installation-guides/restack.md

```diff
@@ -0,0 +1,20 @@
+# Restack
+
+[Restack](https://www.restack.io) is platform to discover and deploy production ready opensource tools in minutes.
+You only need to bring your AWS account and within minutes, your favorite open source tool will be deployed on Kubernetes, on your own cluster.
+Restack is made so that developers can concentrate on coding and don’t have to worry about DevOps. 
+Restack provides auto-scalable, cloud-native, instant, production-ready deployment of open source tools.
+
+## One-Click Deployment
+
+You can deploy Appsmith on the Restack platform with just one click.
+
+The easiest way to do this is
+
+1. [Login to Restack](https://restack.io/login) with your GitHub or email.
+2. Connect your AWS account to create a cluster. 
+3. Search for Appsmith in the restack console and click on add application. 
+
+Here’s a [detailed guide](https://www.restack.io/docs/deploy-appsmith-on-restack#overview) provided in Restack’s docs.  
```

Comment:
@kuberaspeaking - the link (https://www.restack.io/docs/deploy-appsmith-on-restack#overview) gives a 404. Is this expected?

Reply:
The page slug was recently changed. Updating it in the docs.